docs/README.md

```diff
@@ -33,13 +33,47 @@ This library requires `algosdk` as a peer dependency. Ensure you have it install
 To use this library simply include the following at the top of your file:
 
 ```typescript
-import * as algokit from '@algorandfoundation/algokit-utils'
+import algokit from '@algorandfoundation/algokit-utils'
 ```
 
-Then you can use intellisense to auto-complete the various functions that are available by typing `algokit.` in your favourite Integrated Development Environment (IDE), or you can refer to the [reference documentation](code/modules/index.md).
+> [!WARNING]
+> Previous versions of AlgoKit Utils encouraged you to include an import that looks like this (note the subtle difference of the extra `* as`):
+>
+> ```typescript
+> import * as algokit from '@algorandfoundation/algokit-utils'
+> ```
+>
+> This version will still work, but it exposes an older, function-based interface to the functionality that is in the process of being deprecated. The recommended way to use AlgoKit Utils is via the `AlgorandClient` class mentioned below, which is easier and more convenient to use. Some functionality won't yet be migrated to the new approach and this old approach will be needed, but the documentation pages will indicate when this is the case.
```

Comment:
This allows us to a) make a clean break between legacy and new way of using AlgoKit Utils without a breaking change. It also allows us to link to this Usage section across the docs in any part of the docs that is referencing the legacy stuff so we don't have to replace everything in a big bang. We can do a few pull requests one-by-one each each with a methodical focus so we are less likely to get stuff wrong. The focus of this PR is deprecating account, network-client and localnet.